tests fail locally on subsequent runs

### Steps to Reproduce

 - Clone `ocw-studio` on the `master` branch
 - `docker-compose up`
 - Open a new terminal tab
 - `docker-compose run --rm web pytest`
 - `docker-compose run --rm web pytest`

### Expected Behavior

The tests run successfully both times

### Actual Behavior

The tests fail on subsequent runs, outputting 3 failures and 63 errors

### Stacktrace

```
FAILED websites/permissions_test.py::test_create_global_groups - assert (2, 0) == (0, 0)
FAILED websites/views_test.py::test_website_starters_local[True-3] - assert 2 == 3
FAILED websites/views_test.py::test_website_starters_local[False-2] - assert 1 == 2
ERROR ocw_import/tasks_test.py::test_import_ocw2hugo_course_paths[paths0] - websites.models.WebsiteStarter.DoesNotExist: WebsiteStarter matching query does not exist.
ERROR ocw_import/tasks_test.py::test_import_ocw2hugo_course_paths[paths1] - websites.models.WebsiteStarter.DoesNotExist: WebsiteStarter matching query does not exist.
ERROR ocw_import/tasks_test.py::test_import_ocw2hugo_course_paths[None] - websites.models.WebsiteStarter.DoesNotExist: WebsiteStarter matching query does not exist.
ERROR websites/permissions_test.py::test_is_global_admin - django.contrib.auth.models.Group.DoesNotExist: Group matching query does not exist.
ERROR websites/permissions_test.py::test_is_site_admin - django.contrib.auth.models.Group.DoesNotExist: Group matching query does not exist.
ERROR websites/permissions_test.py::test_can_view_edit_preview_website[GET] - django.contrib.auth.models.Group.DoesNotExist: Group matching query does not exist.
ERROR websites/permissions_test.py::test_can_view_edit_preview_website[PATCH] - django.contrib.auth.models.Group.DoesNotExist: Group matching query does not exist.
ERROR websites/permissions_test.py::test_editor_can_view_not_edit_website[PATCH-False] - django.contrib.auth.models.Group.DoesNotExist: Group matching query does not exist.
ERROR websites/permissions_test.py::test_editor_can_view_not_edit_website[GET-True] - django.contrib.auth.models.Group.DoesNotExist: Group matching query does not exist.
ERROR websites/permissions_test.py::test_editor_can_preview_website - django.contrib.auth.models.Group.DoesNotExist: Group matching query does not exist.
ERROR websites/permissions_test.py::test_can_create_website - django.contrib.auth.models.Group.DoesNotExist: Group matching query does not exist.
ERROR websites/permissions_test.py::test_cannot_delete_website - django.contrib.auth.models.Group.DoesNotExist: Group matching query does not exist.
ERROR websites/permissions_test.py::test_author_cannot_view_edit_other_website[GET] - django.contrib.auth.models.Group.DoesNotExist: Group matching query does not exist.
ERROR websites/permissions_test.py::test_author_cannot_view_edit_other_website[PATCH] - django.contrib.auth.models.Group.DoesNotExist: Group matching query does not exist.
ERROR websites/permissions_test.py::test_can_publish_website - django.contrib.auth.models.Group.DoesNotExist: Group matching query does not exist.
ERROR websites/permissions_test.py::test_can_change_collaborators_website[GET] - django.contrib.auth.models.Group.DoesNotExist: Group matching query does not exist.
ERROR websites/permissions_test.py::test_can_change_collaborators_website[PATCH] - django.contrib.auth.models.Group.DoesNotExist: Group matching query does not exist.
ERROR websites/permissions_test.py::test_can_change_collaborators_website[POST] - django.contrib.auth.models.Group.DoesNotExist: Group matching query does not exist.
ERROR websites/permissions_test.py::test_can_change_collaborators_website[DELETE] - django.contrib.auth.models.Group.DoesNotExist: Group matching query does not exist.
ERROR websites/permissions_test.py::test_can_create_website_content - django.contrib.auth.models.Group.DoesNotExist: Group matching query does not exist.
ERROR websites/permissions_test.py::test_can_view_edit_website_content - django.contrib.auth.models.Group.DoesNotExist: Group matching query does not exist.
ERROR websites/permissions_test.py::test_editors_can_delete_own_website_content - django.contrib.auth.models.Group.DoesNotExist: Group matching query does not exist.
ERROR websites/permissions_test.py::test_admins_can_delete_any_website_content - django.contrib.auth.models.Group.DoesNotExist: Group matching query does not exist.
ERROR websites/views_collaborator_test.py::test_websites_collaborators_endpoint_list_permissions - django.contrib.auth.models.Group.DoesNotExist: Group matching query does not exist.
ERROR websites/views_collaborator_test.py::test_websites_collaborators_endpoint_list_permission_denied - django.contrib.auth.models.Group.DoesNotExist: Group matching query does not exist.
ERROR websites/views_collaborator_test.py::test_websites_collaborators_endpoint_list_create - django.contrib.auth.models.Group.DoesNotExist: Group matching query does not exist.
ERROR websites/views_collaborator_test.py::test_websites_collaborators_endpoint_list_create_only_once - django.contrib.auth.models.Group.DoesNotExist: Group matching query does not exist.
ERROR websites/views_collaborator_test.py::test_websites_collaborators_endpoint_list_create_bad_group[global_admin] - django.contrib.auth.models.Group.DoesNotExist: Group matching query does not exist.
ERROR websites/views_collaborator_test.py::test_websites_collaborators_endpoint_list_create_bad_group[global_author] - django.contrib.auth.models.Group.DoesNotExist: Group matching query does not exist.
ERROR websites/views_collaborator_test.py::test_websites_collaborators_endpoint_list_create_bad_group[fake] - django.contrib.auth.models.Group.DoesNotExist: Group matching query does not exist.
ERROR websites/views_collaborator_test.py::test_websites_collaborators_endpoint_list_create_bad_user - django.contrib.auth.models.Group.DoesNotExist: Group matching query does not exist.
ERROR websites/views_collaborator_test.py::test_websites_collaborators_endpoint_detail_create_missing_data[role-Role is required] - django.contrib.auth.models.Group.DoesNotExist: Group matching query does not exist.
ERROR websites/views_collaborator_test.py::test_websites_collaborators_endpoint_detail_create_missing_data[email-Email is required] - django.contrib.auth.models.Group.DoesNotExist: Group matching query does not exist.
ERROR websites/views_collaborator_test.py::test_websites_collaborators_endpoint_detail - django.contrib.auth.models.Group.DoesNotExist: Group matching query does not exist.
ERROR websites/views_collaborator_test.py::test_websites_collaborators_endpoint_detail_denied[get] - django.contrib.auth.models.Group.DoesNotExist: Group matching query does not exist.
ERROR websites/views_collaborator_test.py::test_websites_collaborators_endpoint_detail_denied[patch] - django.contrib.auth.models.Group.DoesNotExist: Group matching query does not exist.
ERROR websites/views_collaborator_test.py::test_websites_collaborators_endpoint_detail_denied[delete] - django.contrib.auth.models.Group.DoesNotExist: Group matching query does not exist.
ERROR websites/views_collaborator_test.py::test_websites_collaborators_endpoint_detail_modify - django.contrib.auth.models.Group.DoesNotExist: Group matching query does not exist.
ERROR websites/views_collaborator_test.py::test_websites_collaborators_endpoint_detail_modify_admin_denied - django.contrib.auth.models.Group.DoesNotExist: Group matching query does not exist.
ERROR websites/views_collaborator_test.py::test_websites_collaborators_endpoint_detail_modify_missing_data - django.contrib.auth.models.Group.DoesNotExist: Group matching query does not exist.
ERROR websites/views_collaborator_test.py::test_websites_collaborators_endpoint_detail_modify_nonadmin_denied - django.contrib.auth.models.Group.DoesNotExist: Group matching query does not exist.
ERROR websites/views_collaborator_test.py::test_websites_collaborators_endpoint_detail_delete - django.contrib.auth.models.Group.DoesNotExist: Group matching query does not exist.
ERROR websites/views_collaborator_test.py::test_websites_collaborators_endpoint_detail_delete_denied - django.contrib.auth.models.Group.DoesNotExist: Group matching query does not exist.
ERROR websites/views_test.py::test_websites_endpoint_list[course] - websites.models.WebsiteStarter.DoesNotExist: WebsiteStarter matching query does not exist.
ERROR websites/views_test.py::test_websites_endpoint_list[None] - websites.models.WebsiteStarter.DoesNotExist: WebsiteStarter matching query does not exist.
ERROR websites/views_test.py::test_websites_endpoint_list_permissions - django.contrib.auth.models.Group.DoesNotExist: Group matching query does not exist.
ERROR websites/views_test.py::test_websites_endpoint_list_create - django.contrib.auth.models.Group.DoesNotExist: Group matching query does not exist.
ERROR websites/views_test.py::test_websites_endpoint_detail[True] - django.contrib.auth.models.Group.DoesNotExist: Group matching query does not exist.
ERROR websites/views_test.py::test_websites_endpoint_detail[False] - django.contrib.auth.models.Group.DoesNotExist: Group matching query does not exist.
ERROR websites/views_test.py::test_websites_endpoint_sorting - websites.models.WebsiteStarter.DoesNotExist: WebsiteStarter matching query does not exist.
ERROR websites/views_test.py::test_website_starters_list - websites.models.WebsiteStarter.DoesNotExist: WebsiteStarter matching query does not exist.
ERROR websites/views_test.py::test_websites_content_list[page] - django.contrib.auth.models.Group.DoesNotExist: Group matching query does not exist.
ERROR websites/views_test.py::test_websites_content_list[] - django.contrib.auth.models.Group.DoesNotExist: Group matching query does not exist.
ERROR websites/views_test.py::test_websites_content_detail - django.contrib.auth.models.Group.DoesNotExist: Group matching query does not exist.
ERROR websites/views_test.py::test_websites_content_delete - django.contrib.auth.models.Group.DoesNotExist: Group matching query does not exist.
ERROR websites/views_test.py::test_websites_content_create - django.contrib.auth.models.Group.DoesNotExist: Group matching query does not exist.
ERROR websites/views_test.py::test_websites_content_create_with_upload - django.contrib.auth.models.Group.DoesNotExist: Group matching query does not exist.
ERROR websites/views_test.py::test_websites_content_edit_with_upload - django.contrib.auth.models.Group.DoesNotExist: Group matching query does not exist.
ERROR websites/views_test.py::test_content_create_page_content[True-True-True] - django.contrib.auth.models.Group.DoesNotExist: Group matching query does not exist.
ERROR websites/views_test.py::test_content_create_page_content[False-True-False] - django.contrib.auth.models.Group.DoesNotExist: Group matching query does not exist.
ERROR websites/views_test.py::test_content_create_page_content[True-False-False] - django.contrib.auth.models.Group.DoesNotExist: Group matching query does not exist.
ERROR websites/views_test.py::test_content_create_page_added_context - django.contrib.auth.models.Group.DoesNotExist: Group matching query does not exist.
ERROR websites/views_test.py::test_websites_content_create_empty - django.contrib.auth.models.Group.DoesNotExist: Group matching query does not exist.
```

--- a/websites/api_test.py
+++ b/websites/api_test.py
@@ -6,7 +6,7 @@ from websites.api import get_valid_new_filename
 from websites.factories import WebsiteContentFactory, WebsiteFactory
 
 
-@pytest.mark.django_db(transaction=True)
+@pytest.mark.django_db
 @pytest.mark.parametrize(
     "existing_filenames,exp_result_filename",
     [

